src/app/login-module/models/userLoginDetails.ts

```diff
@@ -0,0 +1,10 @@
+export class UserLoginDetails {
```

Comment:
Where is login validation response class in this models ?
Every request and response should have separate class 
Never use var classes in angular unless its temporary required.

Reply:
Done